src/components/tagging/form/TagSuggestions.js

```diff
@@ -13,80 +12,41 @@ import type { Team } from '../../../models/Team';
 type Props = {
   activeTeam: ?Team,
   tags: Array<Tag>,
-  addTag: (tag: Tag) => void,
+  onSuggestionClick: (tag: Tag) => void,
 };
 
-type State = {
-  suggestions: Array<{
-    shortcut: string,
-    tag: Tag,
-  }>,
-};
-
-class TagSuggestions extends Component<Props, State> {
-  constructor(props: Props) {
-    super(props);
-
-    this.state = {
-      suggestions: [],
-    };
-  }
-
-  componentDidMount() {
-    window.addEventListener('keypress', this.handleKeyPress);
-    this.fetchSuggestions(this.props.tags);
-  }
-
-  componentDidUpdate(prevProps: Props) {
-    if (!_.isEqual(this.props.tags, prevProps.tags)) {
-      this.fetchSuggestions(this.props.tags);
-    }
-  }
-
-  componentWillUnmount() {
-    window.removeEventListener('keypress', this.handleKeyPress);
-  }
-
-  handleKeyPress = (event: SyntheticKeyboardEvent<HTMLInputElement>) => {
-    const { key } = event;
-    const { suggestions } = this.state;
-    const suggestion = suggestions.find(item => item.shortcut === key);
+function TagSuggestions(props: Props) {
+  const suggestions = useTagSuggestions(props.activeTeam, props.tags);
 
+  const handleKeyPress = (event: SyntheticKeyboardEvent<*>) => {
+    const index = parseInt(event.key, 10) - 1;
+    const suggestion = suggestions[index];
```

Comment:
this is the behavior that prevents using numbers in tags... are you sure this is necessary? Maybe just removing the key press to use suggestion is the better call.

Reply:
Yeah using numbers in tags seems to be desired. 
We could either just kill the shortcuts or use some other combination like pressing multiple keys. But using Alt or Strg or whatever is definitely conflicting with browser or OS shortcuts.
So looks like killing.